app/components/ui/molecules/StaffGrid.md

```diff
@@ -0,0 +1,31 @@
+Staff Grid
+
+```js
+const staff = [
+  {
+    photoURL: 'https://submit.elifesciences.org/images/WeiMun.png',
+    name: 'Wei Mun Chan',
+    jobTitle: 'Editorial Manager',
+    telephone: '+44 [0]1223 855379',
+  },
+  {
+    photoURL: 'https://submit.elifesciences.org/images/HannahD.png',
+    name: 'Hannah Drury',
+    jobTitle: 'Senior Editorial Assistant',
+    telephone: '+44 [0]1223 855348',
+  },
+  {
+    photoURL: 'https://submit.elifesciences.org/images/Maria.png',
+    name: 'Maria Guerreiro',
+    jobTitle: 'Senior Editorial Assistant',
+    telephone: '+44 [0]1223 750665',
+  },
+  {
+    photoURL: 'https://submit.elifesciences.org/images/Susanna.png',
+    name: 'Susanna Richmond',
+    jobTitle: 'Editorial Assistant',
+    telephone: '+44 [0]1223 855377',
+  },
+]
```

Comment:
There's nothing *wrong* with passing an array of JavaScript objects to this component but it's not very *Reacty*. Again, have a look at `TwoColumnLayout` to see how you can pass the data as simple React elements. This also has the advantage that you decouple the content from the layout, allowing the layout to potentially be used with other components.

Reply:
Actually I just noticed `TwoColumnLayout` [can be improved](https://github.com/elifesciences/elife-xpub/pull/865).